web/libs/editor/src/stores/Annotation/Annotation.js

```diff
@@ -659,12 +658,11 @@ const _Annotation = types
 
       self.names.forEach((tag) => tag.needsUpdate && tag.needsUpdate());
       self.updateAppearenceFromState();
-      if (isFF(FF_DEV_2432)) {
-        const areas = Array.from(self.areas.values());
-        const filtered = areas.filter((area) => area.isDrawing);
+      const areas = Array.from(self.areas.values());
+      const filtered = areas.filter((area) => area.isDrawing);
```

Comment:
```suggestion
      // It should find just one unfinished region, but just in case we work with array
      const filtered = areas.filter((area) => area.isDrawing);
```